src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -13,7 +13,7 @@
   --pf-c-dropdown__toggle--BorderWidth: var(--pf-global--BorderWidth--sm);
   --pf-c-dropdown__toggle--BorderTopColor: var(--pf-global--BorderColor--300);
   --pf-c-dropdown__toggle--BorderRightColor: var(--pf-global--BorderColor--300);
-  --pf-c-dropdown__toggle--BorderBottomColor: var(--pf-global--BorderColor--200);
+  --pf-c-dropdown__toggle--BorderBottomColor: var(--pf-global--BorderColor--100);
```

Comment:
Was this on purpose? The border color should match other form control elements. Currently it looks like this:

<img width="271" alt="Screen Shot 2020-03-10 at 5 31 22 PM" src="https://user-images.githubusercontent.com/35148959/76365351-178f5380-62f5-11ea-855d-a187f8de0557.png">